Sources/ObjCSupport/Builders/LOKButtonLayoutBuilder.m

```diff
@@ -10,96 +10,119 @@
     return ^LOKButtonLayoutBuilder *(UIFont * font){
-        self.font = font;
+        self.privateFont = font;
         return self;
     };
 }
 
-- (LOKButtonLayoutBuilder * _Nonnull (^)(UIImage * _Nullable))withImage {
+- (LOKButtonLayoutBuilder * _Nonnull (^)(UIImage * _Nullable))image {
     return ^LOKButtonLayoutBuilder *(UIImage * image){
-        self.image = image;
+        self.privateImage = image;
         return self;
     };
 }
 
-- (LOKButtonLayoutBuilder * _Nonnull (^)(CGSize))withImageSize {
+- (LOKButtonLayoutBuilder * _Nonnull (^)(CGSize))imageSize {
     return ^LOKButtonLayoutBuilder *(CGSize imageSize){
-        self.imageSize = imageSize;
+        self.privateImageSize = imageSize;
         return self;
     };
 }
 
-- (LOKButtonLayoutBuilder * _Nonnull (^)(NSValue * _Nullable))withContentEdgeInsets {
+- (LOKButtonLayoutBuilder * _Nonnull (^)(NSValue * _Nullable))contentEdgeInsets {
     return ^LOKButtonLayoutBuilder *(NSValue * _Nullable contentEdgeInsets){
-        self.contentEdgeInsets = contentEdgeInsets;
+        self.privateContentEdgeInsets = contentEdgeInsets;
         return self;
     };
 }
 
-- (LOKButtonLayoutBuilder * _Nonnull (^)(LOKAlignment * _Nonnull))withAlignment {
+- (LOKButtonLayoutBuilder * _Nonnull (^)(LOKAlignment * _Nonnull))alignment {
     return ^LOKButtonLayoutBuilder *(LOKAlignment * alignment){
-        self.alignment = alignment;
+        self.privateAlignment = alignment;
         return self;
     };
 }
 
-- (LOKButtonLayoutBuilder * _Nonnull (^)(LOKFlexibility * _Nonnull))withFlexibility {
+- (LOKButtonLayoutBuilder * _Nonnull (^)(LOKFlexibility * _Nonnull))flexibility {
     return ^LOKButtonLayoutBuilder *(LOKFlexibility * flexibility){
-        self.flexibility = flexibility;
+        self.privateFlexibility = flexibility;
         return self;
     };
 }
 
-- (LOKButtonLayoutBuilder * _Nonnull (^)(NSString * _Nonnull))withViewReuseId {
+- (LOKButtonLayoutBuilder * _Nonnull (^)(NSString * _Nonnull))viewReuseId {
     return ^LOKButtonLayoutBuilder *(NSString * viewReuseId){
-        self.viewReuseId = viewReuseId;
+        self.privateViewReuseId = viewReuseId;
         return self;
     };
 }
 
-- (LOKButtonLayoutBuilder * _Nonnull (^)(Class _Nonnull))withViewClass {
+- (LOKButtonLayoutBuilder * _Nonnull (^)(Class _Nonnull))viewClass {
     return ^LOKButtonLayoutBuilder *(Class viewClass){
-        self.viewClass = viewClass;
+        self.privateViewClass = viewClass;
         return self;
     };
 }
 
-- (LOKButtonLayoutBuilder * _Nonnull (^)(void(^ _Nullable)(UIButton *_Nonnull)))withConfig {
+- (LOKButtonLayoutBuilder * _Nonnull (^)(void(^ _Nullable)(UIButton *_Nonnull)))config {
     return ^LOKButtonLayoutBuilder *(void(^ _Nullable config)(UIButton *_Nonnull)){
-        self.configure = config;
+        self.privateConfigure = config;
         return self;
     };
 }
 
+- (LOKInsetLayoutBuilder * _Nonnull (^)(EdgeInsets))insets {
```

Comment:
what do you think about renaming `EdgeInsets` and `View` to `LOKEdgeInsets` and `LOKView` in this PR?

Reply:
Yeah. Sure. I can do that.